sdk/hdinsight/Azure.ResourceManager.HDInsight/CHANGELOG.md

```diff
@@ -1,18 +1,14 @@
 # Release History
 
-## 1.2.0-beta.3 (Unreleased)
+## 1.2.0-beta.3 (2024-09-03)
 
 ### Features Added
 
-
+- Enable setting IP tags when creating HDInsight cluster.
+- Enable update the managed identity of the cluster.
+- Enable manage Azure Monitor Agent logs integration on an HDInsight cluster.
```

Comment:
Enable setting --> Support to set
Enable update xx --> Support to update managed identity of cluster.
an-->a